test/proposal-pluginmanager.spec.ts

```diff
@@ -77,7 +77,8 @@ const createAddProposal = async (arc: Arc, dao: DAO, plugin: PluginManagerPlugin
 
   //Wait until plugin is indexed
   await waitUntilTrue(() => newPlugins.length > 0)
-  return newPlugins[0]
+  const proposalData = [newPlugins[0], addProposalStates[0].pluginToRegisterDecodedData.params[2].value]
```

Comment:
why not return the whole addProposalStates[0] here ?

Reply:
please test decode also for `replace` and `remove`  tests

Reply:
If the test checks only the voting params so why return unnecessary data?

Reply:
Okay.

Reply:
> If the test checks only the voting params so why return unnecessary data?

so it can be used to extend the test for other fields as well.